src/modules/app.js

```diff
@@ -34,10 +36,122 @@ const fetchData = async () => {
+      div1.id = `staticBackdrop${category.idCategory}`;
+      div1.setAttribute('data-bs-backdrop', 'static');
+      div1.setAttribute('tabindex', '-1');
+      div1.setAttribute('aria-labelledby', 'exampleModalLabel');
+      div1.setAttribute('aria-hidden', 'true');
+
+      const div2 = document.createElement('div');
+      div2.classList = 'modal-dialog modal-dialog-scrollable modal-xl';
+
+      const div3 = document.createElement('div');
+      div3.classList = 'modal-content';
+
+      const div4 = document.createElement('div');
+      div4.classList = 'modal-body p-3 pt-5 d-flex flex-column align-items-center justify-content-center';
+      div4.style = 'overflow-y:scroll;';
+
+      const button = document.createElement('button');
+      button.type = 'button';
+      button.classList = 'btn-close';
+      button.style = 'position: absolute; top:20px; right:20px;';
+      button.setAttribute('data-bs-dismiss', 'modal');
+      button.setAttribute('aria-lable', 'Close');
+
+      const div5 = document.createElement('div');
+      div5.classList = 'd-flex align-items-start justify-content-between';
+
+      const image = document.createElement('img');
+      image.src = `${category.strCategoryThumb}`;
+      image.alt = `${category.strCategory}`;
+      image.setAttribute('width', '300');
+      image.classList = 'mb-1 pt-5';
+
+      const heading = document.createElement('h3');
+      heading.innerHTML = category.strCategory;
+
+      const para = document.createElement('p');
+      para.style = 'text-align:justify';
+      para.classList = 'px-3';
+      para.innerText = category.strCategoryDescription.substring(0, 206);
+
+      const heading2 = document.createElement('h4');
+      heading2.innerHTML = 'Comments';
+      heading2.id = `commentsHeading${category.idCategory}`;
+
+      const div6 = document.createElement('div');
+      div6.id = `commentList${category.idCategory}`;
+      div6.classList = 'mb-3';
+
+      const heading3 = document.createElement('h4');
+      heading3.innerHTML = 'Add a comment';
+
+      const form = document.createElement('form');
+      form.id = `commentForm${category.idCategory}`;
+
+      const input1 = document.createElement('input');
+      input1.type = 'hidden';
+      input1.name = 'id';
+      input1.value = category.idCategory;
+
+      const input2 = document.createElement('input');
+      input2.type = 'text';
+      input2.placeholder = 'Your Name';
+      input2.classList = 'form-control mb-3';
+      input2.name = 'username';
+
+      const textarea = document.createElement('textarea');
+      textarea.name = 'comment';
+      textarea.rows = '5';
+      textarea.cols = '5';
+      textarea.placeholder = 'Your Insights';
+      textarea.classList = 'form-control mb-3';
+      textarea.style = 'resize:none';
+
+      const button2 = document.createElement('button');
+      button2.classList = 'btn btn-primary mb-3 float-end';
+      button2.innerHTML = 'Comment';
```

Comment:
[optional]Great work so far ❤️ 
Your JS logic is working correctly, but it would be more appropriate if you reconsider the styling of the comment button in order to make it look like the home page buttons. 👍 
![btn-ksks](https://user-images.githubusercontent.com/122353030/233128687-47a67971-aea6-4bc3-a041-dcb1d4dd1ab7.JPG)
is